### ISSUE TITLE
video language endpoint: show progress of completion

### ISSUE THREAD

**Issue by wivaku** [pculture/unisubs#2517]
The [video language endpoint](http://amara.readthedocs.org/en/latest/api.html#video-language-resource) now shows `subtitle_count`. This is the total number of subtitles, copied from the timed source language. So, if the source has 100 subtitles, this target language has 100 subtitles. What we are interested in (in addition): the number of translated subtitles. E.g. 20 (so: 20/100 = 20% complete).

In short: in addition to `subtitle_count`, enable e.g. `subtitle_empty` or `subtitle_completed`.
